bpf/bpf.go

```diff
@@ -1041,6 +1095,19 @@ func (b *BPFLib) RemoveXDP(ifName string, mode XDPMode) error {
 	progName := getProgName(ifName)
 	progPath := filepath.Join(b.xdpDir, progName)
 
+	// Verify Calico XDP program
+	id, err := b.getCalicoBPFProgIds()
+	if err != nil {
+		return err
+	}
+	verify, err := b.canRemoveXDP(ifName, id)
```

Comment:
I'm unclear what the return bool value of `canRemoveXDP` means, but I think it is "There is an XDP program and it's Calico, so please remove it"  Is that right?

Reply:
Basically, yes.

Reply:
Can you add a comment to say that above the definition of `canRemoveXDP`?